<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteWindow</class>
 <widget class="QWidget" name="PaletteWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Палитра</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="lblPalette">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="lblPath">
        <property name="text">
         <string>Путь к палитре:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leditPath">
        <property name="text">
         <string>plugins/palettes/monotonic_cubic_interpolation.png</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBrowse">
        <property name="text">
         <string>Обзор...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QSlider" name="slrShift">
        <property name="maximum">
         <number>1024</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QSpinBox" name="spinShift">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="prefix">
           <string>Сдвиг палитры: </string>
          </property>
          <property name="maximum">
           <number>1024</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnReset">
          <property name="text">
           <string>Сброс</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnOk">
          <property name="text">
           <string>Ок</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>PaletteWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinShift</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slrShift</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slrShift</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinShift</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
